src/vsr/replica_test.zig

```diff
@@ -966,39 +969,6 @@ test "Cluster: sync: partition, lag, sync (transition from idle)" {
     }
 }
 
-test "Cluster: sync: sync, bump target, sync" {
-    const t = try TestContext.init(.{ .replica_count = 3 });
-    defer t.deinit();
-
-    var c = t.clients(0, t.cluster.clients.len);
-
-    t.replica(.R2).drop_all(.R_, .bidirectional);
-    try c.request(checkpoint_2_trigger, checkpoint_2_trigger);
-
-    // Allow R2 to complete SyncStage.requesting_target, but get stuck
-    // during SyncStage.requesting_checkpoint.
-    t.replica(.R2).pass_all(.R_, .bidirectional);
-    t.replica(.R2).drop(.R_, .outgoing, .request_sync_checkpoint);
-    t.run();
-    try expectEqual(t.replica(.R2).sync_status(), .requesting_checkpoint);
-    try expectEqual(t.replica(.R2).sync_target_checkpoint_op(), checkpoint_2);
-
-    // R2 discovers the newer sync target and restarts sync.
-    try c.request(checkpoint_3_trigger, checkpoint_3_trigger);
-    try expectEqual(t.replica(.R2).sync_status(), .requesting_checkpoint);
-    try expectEqual(t.replica(.R2).sync_target_checkpoint_op(), checkpoint_3);
-
-    t.replica(.R2).pass(.R_, .bidirectional, .request_sync_checkpoint);
-    t.run();
-
-    try expectEqual(t.replica(.R_).status(), .normal);
-    try expectEqual(t.replica(.R_).commit(), checkpoint_3_trigger);
-    try expectEqual(t.replica(.R_).sync_status(), .idle);
-
-    t.run(); // (Wait for grid sync to finish.)
-    try TestReplicas.expect_sync_done(t.replica(.R_));
-}
-
```

Comment:
We no longer have this bump sync target case at all, updates are _mostly_ atomic.